indico/modules/events/contributions/templates/forms/contribution_person_widget.html

```diff
@@ -0,0 +1,226 @@
+            {% endif %}
+        </div>
+        <div class="i-box-footer" style="text-align: right;">
+            <i class="info-helper"
+               title="{% trans %}Added people can be modified moving the cursor over them.{% endtrans %}"></i>
+            <span id="add-existing-{{ field.id }}" class="i-button highlight">
+                {% trans %}Choose{% endtrans %}
+            </span>
+            <span id="add-new-{{ field.id }}" class="i-button"
+                  title="{% trans %}Enter a person manually{% endtrans %}">
+                {% trans %}Enter{% endtrans %}
+            </span>
+        </div>
+    </div>
+{% endblock %}
+
+{% block javascript %}
+    <script>
+        $(function() {
+            'use strict';
+
+            var $field = $('#{{ field.id }}');
+            var $fieldDisplay = $('#people-list-{{ field.id }}');
+            var $authorList = $fieldDisplay.find('#author-list-{{ field.id }}');
+            var $noAuthorPlaceholder = $fieldDisplay.find('#no-author-placeholder-{{ field.id }}');
+            var $coauthorList = $fieldDisplay.find('#coauthor-list-{{ field.id }}');
+            var $coauthorListTitle = $fieldDisplay.find('#coauthor-list-title-{{ field.id }}');
+            var $noCoauthorPlaceholder = $fieldDisplay.find('#no-coauthor-placeholder-{{ field.id }}');
+            var $otherList = $fieldDisplay.find('#other-list-{{ field.id }}');
+            var $otherListTitle = $fieldDisplay.find('#other-list-title-{{ field.id }}');
+            var $nobodyPlaceholder = $fieldDisplay.find('#nobody-placeholder-{{ field.id }}');
+            var $buttonAddExisting = $('#add-existing-{{ field.id }}');
+            var $buttonAddNew = $('#add-new-{{ field.id }}');
+
+            function renderPeople(people) {
+                $authorList.html('');
+                $coauthorList.html('');
+                $otherList.html('');
+                _.each(people, renderPerson);
+                $noAuthorPlaceholder.toggle($authorList.is(':empty'));
+                {% if not field.show_empty_coauthors %}
+                    $coauthorListTitle.toggle($coauthorList.is(':not(:empty)'));
+                {% else %}
+                    $noCoauthorPlaceholder.toggle($coauthorList.is(':empty'));
+                {% endif %}
+                $otherListTitle.toggle($otherList.is(':not(:empty)'));
+                $nobodyPlaceholder.toggle(people.length == 0);
+            }
+
+            function renderPerson(person) {
+                var $personRow = $('<div>').addClass('person-row');
+                var $personName = $('<span>').text(person.name);
+                var $personStatus = $('<span>').addClass('person-status');
+                var $personRoles = $('<span>').addClass('person-roles');
+                var $personButtons = $('<span>').addClass('person-buttons');
+                var $buttonRemove = $('<a>').addClass('i-button-icon danger icon-close').attr('title', $T("Remove person"));
```

Comment:
I think `addClass` and `attr` are much cleaner interfaces, but if you all have a strong opinion on this I have no problem changing all constructors.